package/contents/ui/config/ConfigActions.qml

```diff
@@ -0,0 +1,60 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+import org.kde.kirigami as Kirigami
+import org.kde.kcmutils as KCM
+import org.kde.plasma.core as PlasmaCore
+
+import "../lib"
+
+KCM.SimpleKCM {
+	
+	property alias cfg_clickCommand: clickCommand.text
+	property alias cfg_mousewheelUpCommand: mousewheelUpCommand.text
+	property alias cfg_mousewheelDownCommand: mousewheelDownCommand.text
```

Comment:
Hmm,  breaking the the single config page up into 3 different pages might be neat since I kept adding more stuff below the fold.

----

~~I need to check if the `ConfigActions.qml` populates config values from the `General` tab.~~ Edit: Looks like it does. This is probably a myth that I made up and keep needing to check.

Here's the code that wraps these config pages if you're interested:

* https://invent.kde.org/plasma/plasma-desktop/blob/master/desktoppackage/contents/configuration/AppletConfiguration.qml#L109
* https://invent.kde.org/plasma/plasma-desktop/blob/master/desktoppackage/contents/configuration/AppletConfiguration.qml#L48